notebooks/confirm_gpu_available.ipynb

```diff
@@ -16,6 +16,12 @@
     "pip3 install torch==1.9.0+cu111 torchvision==0.10.0+cu111 torchaudio==0.9.0 -f https://download.pytorch.org/whl/torch_stable.html\n",
     "```\n",
     "\n",
+    "If you are running an instance with jupyerlab >= 3.1.0 and jupyterlab-requirements >= 0.11.0 you can install the requirements running the following command in a terminal since they are also stored in this notebooks metadata. \n",
```

Comment:
```suggestion
    "If you are running an instance with jupyterlab >= 3.1.0 and jupyterlab-requirements >= 0.11.0 you can install the requirements running the following command in a terminal since they are also stored in this notebooks metadata. \n",
```